sphinx/analysis.rst

```diff
@@ -0,0 +1,36 @@
+.. methods:
+
+Analysis Methods
+################
+
+Analysis Methods work on PlantData objects to produce high level analysis products, such as the long term AEP.
+These methods rely on the more generic toolkit modules, chaining them together to create reproducible analysis processes.
+
+Plant Level Analysis
+********************
+
```

Comment:
I don’t think `MonteCarloAEPResult` and `get_annual_values()` should be listed at the same level as `MonteCarloAEP` and `EYAGapAnalysis`, since they are there to support the main analysis methods (specifically, I'm referring to how they appear on the table of contents on the right side of the page when the docs are built)